apps/condo/domains/news/tasks/notifyResidentsAboutNewsItem.spec.js

```diff
@@ -99,8 +107,8 @@ describe('notifyResidentsAboutNewsItem', () => {
                         })],
                     }),
                     meta: expect.objectContaining({
-                        title: 'Lorem ipsum dolor sit amet, consectetur...',
-                        body: 'Commodo viverra maecenas accumsan lacus vel facilisis volutpat est velit. Et malesuada fames ac turpis egestas sed tempus urna et. At augue eget...',
+                        title: truncate(newsItemTitle, { length: TITLE_MAX_LEN, separator: ' ', omission: '...' }),
```

Comment:
is truncate really not necessary here?

Reply:
Really needed, because I changed hard-code above using faker. And variable should truncate before testing